conf/test_full.config

```
@@ -0,0 +1,22 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running full-size tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a full size pipeline test. Use as follows:
+ *   nextflow run nf-core/rnafusion -profile test_full,<docker/singularity>
+ */
+
+params {
+  config_profile_name = 'Full test profile'
+  config_profile_description = 'Full test dataset to check pipeline function'
+
+  // Input data for full size test
+  // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
+  // TODO nf-core: Give any required params for the test so that command line flags are not needed
+  single_end = false
+  input_paths = [
+    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R1.tiny.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R2.tiny.fastq.gz']],
+    ['SRR389222', ['https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz']]
+  ]
```

We actually have test dataset from `FusionCatcher` here: https://github.com/nf-core/test-datasets/tree/rnafusion
```suggestion
input_paths = [
    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/rnafusion/testdata/human/reads_1.fq.gz', 'https://github.com/nf-core/test-datasets/raw/rnafusion/testdata/human/reads_2.fq.gz']]
  ]
```

---

I was more thinking into looking into that for another PR

---

Ok, let's get back to this later